apps/public-docsite-v9/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "just-scripts build",
     "build-storybook": "build-storybook -s ./public -o ./dist/storybook --docs",
-    "chromatic": "npx chromatic@6.4.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name build-storybook",
+    "chromatic:branch": "npx chromatic@6.4.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name build-storybook",
+    "chromatic": "npx chromatic@6.4.3 --project-token 15c373f76c3a --exit-zero-on-changes --build-script-name build-storybook --branch-name microsoft:master",
```

Comment:
Should this also use `$CHROMATIC_PROJECT_TOKEN`?

Reply:
fixed and new key generated